style/styles.css

```diff
@@ -594,6 +603,95 @@ textarea {
     margin-bottom: 100px;
   }
 
+  /* pop project details */
+  .projectDetail {
+    display: flex;
+    position: fixed;
+    justify-content: center;
+    top: 0;
+    right: 0;
+    width: 100%;
+    max-height: 100vh;
+    z-index: 99;
+    margin-top: 0;
+    background-color: #fff;
+    overflow-y: auto;
+    box-sizing: border-box;
+  }
```

Comment:
- [ ] Please kindly add margin to your popup window for both desktop and mobile version. Part of our duty as developers is our capability to reproduce any design given to us expected. 👍 

## Figma design for desktop version

![q2](https://user-images.githubusercontent.com/103386183/209178472-b82299a7-fd6a-41b8-8546-d75dccd07dc7.png)

## Figma design for mobile version
![q3](https://user-images.githubusercontent.com/103386183/209178548-c8c1f273-9285-4eff-af4b-3847e8d38b4f.png)